extensions/resource-deployment/src/interfaces.ts

```diff
@@ -244,11 +261,13 @@ export enum FieldType {
 	Options = 'options',
 	RadioOptions = 'radio_options',
 	ReadonlyText = 'readonly_text',
+	SummaryText = 'summary_text',
 	Checkbox = 'checkbox',
 	AzureAccount = 'azure_account',
 	AzureLocations = 'azure_locations',
 	FilePicker = 'file_picker',
-	KubeClusterContextPicker = 'kube_cluster_context_picker'
+	KubeClusterContextPicker = 'kube_cluster_context_picker',
+	LinkedText = 'linked_text'
```

Comment:
I don't think we need this, you should be able to just extend the ReadonlyText.

Reply:
i am still not convinced, but this is not a big deal. is the name proper? should it be hyperlink_text?

Reply:
Can i keep the feature only in linked_text and remove from the readonly_text. Readonly_text is overloaded with linking functionality which makes usage harder.

Reply:
Can i keep the feature only in linked_text and remove from the readonly_text. Readonly_text is overloaded with linking functionality which makes usage harder.

Reply:
@ranasaria , I don't think it is necessary, to make it easier, we just need to add a sample in: samples/sample-resource-deployment

, btw, please add a sample for your new json based wizard.

Reply:
I will add a sample for thee new JSON based wizard in samples/sample-resource-deployment. After chatting offline with alan, we decided to remove the linking feature from readonly_text FieldType so that readonly_text mainly does only the thing of allowing a user to put two different TextComponents as a field.

Reply:
I will also rename 'linked_text' to 'hyperlinked_text'

Reply:
I really don't understand the need for so many text fields. Why isn't there just one text field that has properties for the different types, such as isReadOnly, links, etc...

Maybe a bit late to be changing it here - but let's try to make sure we're thinking of stuff like this in the future when we're doing work like this so we don't end up with a huge set of components that are all basically doing the same thing.

Reply:
Consolidate ReadOnlyText, EvaluatedText, and HyperlinkedText into a single FieldType of ReadOnlyText and also consolidated dropdown options and radioOptions into a single FieldType